test/errors-spec.js

```diff
@@ -178,6 +178,17 @@ describe("Parser:", function() {
         });
     });
 
+    describe("#verb", function() {
+        it("complains about mismatched \\verb with end of string", function() {
+            expect("\\verb|hello").toFailWithParseError(
+                "\\verb ended by end of line instead of matching delimiter");
+        });
+        it("complains about mismatched \\verb with end of line", function() {
+            expect("\\verb|hello\nworld|").toFailWithParseError(
+                "\\verb ended by end of line instead of matching delimiter");
+        });
```

Comment:
Is there a way to test the exception being throw in Parser.js?

Reply:
I'm confused -- isn't that what this is doing?  Or do you mean confirming that the error is actually of type `ParseError`?  We could probably rewrite `toFailWithParseError` to do this.

Reply:
This is the error that's being thrown in `functions.js`.  The error thrown in `Parser.js` has the text `"\\verb failure to parse"`.

Reply:
From your comment above it sounds like it should never happen though, so I guess we can't test for it.

Reply:
Oh, I see.  As described above, that error can never be thrown.